apis/infix-en_GB/extensions/jdk8/atrium-api-infix-en_GB-jdk8-jvm/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/jdk8/PathFeatureAssertionsSpec.kt

```diff
@@ -34,5 +34,7 @@ class PathFeatureAssertionsSpec : ch.tutteli.atrium.specs.integration.PathFeatur
 
         a1.parent
         a1 parent {}
+
+        a1.resolve("test")
```

Comment:
don't use parentheses here. That's the main difference between the fluent and the infix api
```suggestion
        a1 resolve "test"
```